tests/analysis/test_rdf.py

```diff
@@ -0,0 +1,28 @@
+"""Pytest for dynsight.analysis.compute_rdf."""
+
+from pathlib import Path
+
+import MDAnalysis
+import numpy as np
+
+from dynsight.analysis import compute_rdf
+
+
+def test_compute_rdf() -> None:
+    original_dir = Path.cwd()
```

Comment:
I would use `pathlib.Path(__file__).resolve().parent` to resolve the file's parent directory.

Reply:
do you mean instead of Path.cwd()?

Reply:
Correct: 
```python
script_path =  pathlib.Path(__file__).resolve().parent
```
instead of `original_dir` and then expand from that